lib/view/main_view/home/send_page.dart

```diff
@@ -0,0 +1,228 @@
+import 'package:flutter/material.dart';
+import 'package:wallet_app/model/wallet_info.dart';
+import 'package:wallet_app/view_model/main_model.dart';
+
+/**
+ * 钱包转账
+ */
+class WalletSend extends StatefulWidget {
+  @override
+  _WalletSendState createState() => _WalletSendState();
+}
+class _WalletSendState extends State<WalletSend> {
+
+  MainStateModel stateModel = null;
+  WalletInfo walletInfo;
+  AccountInfo accountInfo;
+  final key = new GlobalKey<ScaffoldState>();
+
+
+  var minerFee = 0.0001;
+  var _feeGroup = 0;
+  void _setvalue2(int value) => setState(() {
+      if(value==0){
+        this.minerFee = 0.0001 ;
+      }
+      if(value==1){
+        this.minerFee = 0.0002 ;
+      }
+      if(value==2){
+        this.minerFee = 0.0003 ;
+      }
+      if(value==-1){
+        this.minerFee = 0.0004 ;
+      }
+
+      _feeGroup = value;
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    stateModel = MainStateModel().of(context);
+    walletInfo = stateModel.currWalletInfo;
+    accountInfo = stateModel.currAccountInfo;
+    return Scaffold(
+        key: this.key,
+        appBar: AppBar(title: Text(accountInfo.name+"转账"),),
+        body: this.body()
+    );
+  }
+
+  Widget makeRadioTiles() {
+    List<Widget> list = new List<Widget>();
+    list.add(RadioListTile<int>(value: 0,title: Text('慢速    0.00001 btc'), groupValue: _feeGroup, onChanged: _setvalue2));
```

Comment:
This is ok to get started for the moment, but be advised the using a 'static' fee list is not advised. Fees should be calculated dynamically based on the number of utxo's the sending address is using as well as the current state of the network.   Using static fees can lead to potentially underfunded addresses in some cases and a transaction that either never confirms or takes too long to confirm

Reply:
Added comment for the code.